dev-util/intel-ocl-sdk/intel-ocl-sdk-18.1.0.015.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit rpm
+
+DESCRIPTION="Intel's implementation of the OpenCL standard"
+HOMEPAGE="https://www.intel.com/content/www/us/en/developer/articles/tool/opencl-drivers.html#cpu-section"
+SRC_URI="https://registrationcenter-download.intel.com/akdlm/irc_nas/vcp/15532/l_opencl_p_${PV}.tgz"
+
+LICENSE="Intel-SDP"
```

Comment:
Do we need to update the LICENSE / add a new one? You mentioned it's changed

Reply:
I think it just needs to be updated. The one currently in the tree is from 2015.

```
End User License Agreement for the Intel(R) Software Development Products
(Version May 2018)
```

There are other licenses: BSD 3-clause, Apache-2, Khronos (probably does not apply since KHR headers are not included), LLVM (probably covers `libclang` and a few others).

Reply:
Oh I see, so we always needed bindist and such. Could you add the other needed licenses and update the licence file in this PR too (given we're going to cleanup old immediately as it's so broken)?